tests/RobotFramework/tests/cumulocity/self-update/tedge-fixed-version.conf

```diff
@@ -0,0 +1,31 @@
+Package: tedge-full
+Pin: version 1.0.0
+Pin-Priority: 1001
+
+Package: tedge
+Pin: version 1.0.0
+Pin-Priority: 1001
+
+Package: tedge-mapper
+Pin: version 1.0.0
+Pin-Priority: 1001
+
+Package: tedge-agent
+Pin: version 1.0.0
+Pin-Priority: 1001
+
+Package: c8y-firmware-plugin
+Pin: version 1.0.0
+Pin-Priority: 1001
+
+Package: c8y-remote-access-plugin
+Pin: version 1.0.0
+Pin-Priority: 1001
+
+Package: tedge-apt-plugin
+Pin: version 1.0.0
+Pin-Priority: 1001
+
+Package: tedge-watchdog
+Pin: version 1.0.0
+Pin-Priority: 1001
```

Comment:
There is no end-of-line before the EOF. Can this confuse `apt`?

Reply:
I would be surprised since it is not toml ;) But I will double check and post back here.

Reply:
This comment has now been removed because an alternative approach which avoids using package pinning was used. 3f1d1ffd1670f0c560d5cd77dc7d39a8d5a52c04